finishFeature/src/main/java/com/panhuk/finishfeature/FinishFragment.kt

```diff
@@ -0,0 +1,109 @@
+package com.panhuk.finishfeature
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Button
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.fragment.app.Fragment
+import com.panhuk.core.RIGHT_ANSWERS
+import com.panhuk.core.TOTAL_ANSWERS
+import com.panhuk.core.Screen
+import com.panhuk.finishfeature.databinding.FinishFragmentBinding
+
+class FinishFragment : Fragment() {
+  private var _binding: FinishFragmentBinding? = null
+  private val binding get() = _binding!!
```

Comment:
You can put everything related to the ViewBinding in the base class. It will allow hiding 90% of the boilerplate code of it.

Reply:
Yes, also good idea I am planning to make this basic class when I close all the PR's so I can refactor all the code with above approach